aws/v11.4.1/release.yaml

```
@@ -0,0 +1,59 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  name: v11.4.1
+spec:
+  state: active
+  date: 2020-07-02T15:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - name: cert-manager
+    componentVersion: 0.9.0
+    version: 1.0.8
+  - name: chart-operator
+    version: 0.12.4
+  - name: cluster-autoscaler
+    componentVersion: 1.16.5
+    version: 1.16.0
+  - name: coredns
+    componentVersion: 1.6.5
+    version: 1.1.10
+  - name: external-dns
+    componentVersion: 0.7.2
+    version: 1.2.2
+  - name: kiam
+    componentVersion: 3.5.0
+    version: 1.2.2
```

I updated the kiam-app to v1.3.0, see https://github.com/giantswarm/kiam-app/releases/tag/v1.3.0. I would be happy if we use in 11.4.1

```suggestion
    version: 1.3.0
```

---

Fine if you can help with testing - this release has already gotten a bunch more changes. So if you could verify that it works, then fine by me.

---

Sure, I'll help as much as I can 👍